adding_species_manuscript.tex

```diff
@@ -710,6 +728,11 @@ \subsection*{Filling out the missing pieces}
 			will affect the impact of selection, linkage, and linked selection
 			\\
 			\hline
+			Gene conversion parameters &
+			borrow from closest relative with a citeable rate &
+			will affect the impact of selection, linkage, and linked selection
+			\\
+			\hline
 			Demographic model &
```

Comment:
I'll add this to the table, but will mention that this is relevant only if modeling gene conversion is central to the species evolution (such as in bacteria and archea).